src/bot.js

```diff
@@ -41,8 +41,15 @@ bot.on('message', async (message) => {
                 .setFooter('testing the footer');
             message.channel.send(embed);
         }
+
+        if (command === 'help'){
+            embed.setColor(0xffff00)
+                .setTitle('How to use the TriviaBot') 
+                .setDescription('Useful Commands \n-help: Display all the commands \n-play: Initialize the TriviaBot');
+            message.channel.send(embed);
+        }
     }
-    if (message.content === 'anything') {
+    if (message.content === 'anything') { //anything works without -play **bug**
```

Comment:
Good point! We're def gonna get rid of this as soon as we can get a 'real' game going. Thanks for leaving a note!